sympy/printing/latex.py

```diff
@@ -1462,6 +1462,10 @@ def _print_Transpose(self, expr):
         else:
             return "%s^T" % self._print(mat)
 
+    def _print_Trace(self, expr):
+        mat = expr.arg
+        return r"\operatorname{Tr} %s" % self._print(mat)
```

Comment:
I think it would be better to use `\mathrm{Tr}\left (%s \right )`.

Reply:
Should it be `tr` with a lowercase `t`? That's what the pretty printer uses, and it's how Wikipedia renders it https://en.wikipedia.org/wiki/Trace_(linear_algebra) (Wikipedia also uses the parentheses everywhere).

Reply:
OK, that makes sense to me, for consistency. (http://mathworld.wolfram.com/MatrixTrace.html capitalizes, so it seems fairly arbitrary).